Makefile

```diff
@@ -0,0 +1,23 @@
+DEFAULT_GOAL: all
+
+all: lint test build deploy-dev
+
+.PHONY: build
+build:
+	wasm-pack build --target web securedrop-source
+
+.PHONY: deploy-dev
+deploy-dev:
+	@test -d ../securedrop/ || { echo "ERROR: Missing securedrop repo at ../securedrop/" && exit 1 ; }
+	@cp -v -t ../securedrop/securedrop/static/js/ \
```

Comment:
`cp -t` is a GNU-ism that's not supported by BSD `cp` (in my case, on macOS):


```sh-session
$ make all
[...]
cp: illegal option -- t
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file ... target_directory
make: *** [deploy-dev] Error 64
```

It looks like the less-clear latter syntax `cp [...] source_file ... target_directory` is also supported by [GNU `cp`](http://man.he.net/?topic=cp&section=all):

```
       cp [OPTION]... SOURCE... DIRECTORY
```

Reply:
Thank you for testing on macOS! I need a linter that checks are-we-posix-yet. :upside_down_face: Fixed!